.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Order is important. The last matching pattern has the most precedence.
 
-*            @chef/chef-infra-reviewers @chef/chef-infra-approvers @chef/chef-infra-owners
+*            @chef/chef-infra-reviewers @chef/chef-infra-approvers @chef/chef-infra-owners @chef/chef-workstation-approvers
```

Comment:
You want to add all 3 groups

Reply:
Make sure to PR this repo into this list too https://github.com/chef/chef-oss-practices/blob/main/projects/chef-workstation.md

Reply:
If Workstation team is going to take on this repo then Infra should probably get moved off as owners as well.

Reply:
done, created https://github.com/chef/chef-oss-practices/pull/354 too